mangaki/mangaki/tests/test_anidb.py

```diff
@@ -4,15 +4,19 @@
 from django.conf import settings
 from django.test import TestCase
 
-from mangaki.models import Category, Editor, Studio, Work
+from mangaki.models import Category, Editor, Studio, Work, Language, WorkTitle
 from mangaki.utils.anidb import AniDB
 
 
 class AniDBTest(TestCase):
     @staticmethod
-    def create_anime(**kwargs):
-        anime = Category.objects.get(slug='anime')
-        return Work.objects.create(category=anime, **kwargs)
+    def create_anime(work, work_titles):
```

Comment:
See comment on `add_anidb.create_anime` above; let's get rid of this and do this work in `Anidb.get_mangaki_work` instead.